torchao/prototype/inductor/codegen/cpp_int8_sdpa_template.py

```diff
@@ -239,12 +245,12 @@
       long col = 0;
       for (; col < vec_size * (kvBlockSize / vec_size); col += vec_size) {
         auto tmp0 = at::vec::Vectorized<float>::loadu(tmp_in + col);
-        auto tmp1 = tmp0 * vec_sum_scale;
-        auto tmp2 = tmp1.round();
-        auto tmp3 = tmp2 + vec_beta1;
+        auto tmp1 = at::vec::fmadd(tmp0, vec_sum_scale, vec_beta1);
+        auto tmp3 = tmp1.round();
```

Comment:
Could we also apply the optimization to the below masked vectorization part?

Reply:
Updated. Thanks.